templates/compose/cloudreve.yaml

```
@@ -0,0 +1,57 @@
+# documentation: https://docs.cloudreve.org/
+# slogan: A self-hosted file management and sharing system.
+# category: storage
+# tags: file sharing,cloud storage,self-hosted,open-source
+# logo: svgs/cloudreve.svg
+# port: 5212
+
+services:
+  cloudreve:
+    image: cloudreve/cloudreve:latest
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

```suggestion
    image: cloudreve/cloudreve:4.8.0
```